roles/configure-zookeeper/README.md

```diff
@@ -0,0 +1,38 @@
+Role Name
```

Comment:
plz kindly update README , this content is autogenerated ;)

Reply:
working on it

Reply:
updated readme